MREUnityRuntime/MREUnityRuntimeLib/Messaging/Payloads/NetworkCommandPayloads.cs

```diff
@@ -231,14 +231,14 @@ public class UpdateSubscriptions : NetworkCommandPayload
         public Guid Id { get; set; }
 
         /// <summary>
-        /// The subscription types to add to the object. See <see cref="SubscriptionType"/>.
+        /// The subscription types to add to the object. See <see cref="ActorComponentType"/>.
```

Comment:
Might want to update these to "The actor component type to subscribe/unsubscribe to."